app/views/case_workers/admin/case_workers/index.html.haml

```diff
@@ -27,13 +27,10 @@
             = govuk_table_td('data-label': t('.surname')) { case_worker.user.last_name }
 
             = govuk_table_td('data-label': t('.name')) { case_worker.user.first_name }
-
-            = govuk_table_td('data-label': t('.location')) { case_worker.location.name }
+            = govuk_table_td('data-label': t('.status')) { govuk_tag_active_user?(case_worker) }
```

Comment:
I get an error when I run this locally. I think the problem is this line.

`govuk_tag_active_user?(case_worker)` results in `enabled?` being called on `case_worker`, which causes a failure. This is because `case_worker` is an instance of `Caseworker`, which does not define `enabled?`. That method actually expects an instance of `User` to be passed to it, which you can do like this:

```suggestion
            = govuk_table_td('data-label': t('.status')) { govuk_tag_active_user?(case_worker.user) }
```

Reply:
Thanks @mpw5. That makes sense! :)